dev_tools/rubocop/base.yml

```diff
@@ -1,3 +1,7 @@
+require:
+  - rubocop-performance
+  - rubocop-thread_safety
+
```

Comment:
I forgot to create a PR for this when I first found the error. Can you add this to `base.yml`?

``` yaml
inherit_mode:
  merge:
    - Exclude
```

The normal behavior is to override previous rules. For example this repo's [Metrics/BlockLength excludes](https://github.com/generalassembly/shared_configs/blob/master/dev_tools/rubocop/base.yml#L12) are currently being replaced by Core's [Metrics/BlockLength excludes](https://github.com/generalassembly/core/blob/master/.rubocop.yml#L12).